src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -0,0 +1,134 @@
+    end
+    nn_index = MOI.add_constraint(model, f_new, MOI.Nonnegatives(MOI.output_dimension(f_new)))
+    return NormInfinityBridge{T, F}(nn_index)
+end
+
+MOI.supports_constraint(::Type{NormInfinityBridge{T}}, ::Type{<:MOI.AbstractVectorFunction}, ::Type{MOI.NormInfinityCone}) where T = true
+MOIB.added_constrained_variable_types(::Type{<:NormInfinityBridge}) = Tuple{DataType}[]
+MOIB.added_constraint_types(::Type{NormInfinityBridge{T, F}}) where {T, F} = [(F, MOI.Nonnegatives)]
+function concrete_bridge_type(::Type{<:NormInfinityBridge{T}}, H::Type{<:MOI.AbstractVectorFunction}, ::Type{MOI.NormInfinityCone}) where T
+    F = MOIU.promote_operation(+, T, H, H)
+    return NormInfinityBridge{T, F}
+end
+
+# Attributes, Bridge acting as a model
+MOI.get(b::NormInfinityBridge{T, F}, ::MOI.NumberOfConstraints{F, MOI.Nonnegatives}) where {T, F} = 1
+MOI.get(b::NormInfinityBridge{T, F}, ::MOI.ListOfConstraintIndices{F, MOI.Nonnegatives}) where {T, F} = [b.nn_index]
+
+# References
+MOI.delete(model::MOI.ModelLike, c::NormInfinityBridge) = MOI.delete(model, c.nn_index)
+
+# Attributes, Bridge acting as a constraint
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintPrimal, c::NormInfinityBridge)
+    ge_value = MOI.get(model, MOI.ConstraintPrimal(), c.nn_index)
+    t = sum(ge_value) / length(ge_value)
+    d = div(length(ge_value), 2)
+    x = (ge_value[(d + 1):end] - ge_value[1:d]) / 2
+    return vcat(t, x)
+end
+# Given a_i is dual on t - x_i >= 0 and b_i is dual on t + x_i >= 0,
+# the dual on (t, x) in NormInfinityCone is (u, v) in NormOneCone, where
+# v_i = -a_i + b_i and u = sum(a) + sum(b).
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintDual, c::NormInfinityBridge)
+    ge_dual = MOI.get(model, MOI.ConstraintDual(), c.nn_index)
+    t = sum(ge_dual)
+    d = div(length(ge_dual), 2)
+    x = (ge_dual[(d + 1):end] - ge_dual[1:d])
+    return vcat(t, x)
+end
+
+"""
+    NormOneBridge{T}
+
+The `NormOneCone` is representable with LP constraints, since
+``t \\ge \\sum_i |x_i|`` if and only if there exists a vector y such that
+``t \\ge \\sum_i y_i`` and ``y_i \\ge x_i``, ``y_i \\ge -x_i`` for all ``i``.
+"""
+struct NormOneBridge{T, F, G} <: AbstractBridge
+    y::Vector{MOI.VariableIndex}
+    ge_index::CI{F, MOI.GreaterThan{T}}
+    nn_index::CI{G, MOI.Nonnegatives}
+end
+function bridge_constraint(::Type{NormOneBridge{T, F, G}}, model::MOI.ModelLike, f::MOI.AbstractVectorFunction, s::MOI.NormOneCone) where {T, F, G}
+    if f isa MOI.VectorOfVariables # TODO currently needed for test to pass
+        f = MOI.VectorAffineFunction{T}(f)
+    end
+    f_scalars = MOIU.eachscalar(f)
+    d = MOI.dimension(s)
+    y = MOI.add_variables(model, d - 1)
+    ge_index = MOIU.normalize_and_add_constraint(model, MOIU.operate(-, T, f_scalars[1], MOIU.operate(sum, T, y)), MOI.GreaterThan(zero(T)), allow_modify_function=true)
+    lb = f_scalars[2:d]
+    ub = MOIU.operate(-, T, lb)
+    MOIU.operate!(+, T, lb, MOI.VectorOfVariables(y))
```

Comment:
`operate!` is not guarnteed to store the result in the first arg. For instance, it is not the case for VectorOfVariables hence the bug you have (I guess it's the issue).
You need to catch the output of `operate!` in another variable and not use `lb` anymore (as it might or might not have been modified, you don't know).

Reply:
ok that's a little confusing. will try it.

Reply:
indeed that fixed it